frontend/src/features/properties/map/acquisition/AcquisitionContainer.tsx

```diff
@@ -77,17 +81,35 @@ export const AcquisitionContainer: React.FunctionComponent<IAcquisitionContainer
     setContainerState({ selectedMenuIndex: selectedIndex });
   };
 
+  const onSuccess = () => {
+    fetchAcquisitionFile();
+    search();
+  };
+
   // UI components
   const formTitle = containerState.isEditing ? 'Update Acquisition File' : 'Acquisition File';
 
   const menuItems =
-    acquisitionFile?.acquisitionProperties?.map(x => getAcquisitionPropertyName(x).value) || [];
+    acquisitionFile?.fileProperties?.map(x => getAcquisitionPropertyName(x).value) || [];
   menuItems.unshift('File Summary');
 
   if (acquisitionFile === undefined && loadingAcquisitionFile) {
     return <LoadingBackdrop show={true} parentScreen={true}></LoadingBackdrop>;
   }
 
+  if (containerState.activeEditForm === EditFormNames.propertySelector && acquisitionFile) {
+    return (
+      <UpdateProperties
+        file={acquisitionFile}
+        setIsShowingPropertySelector={() =>
+          setContainerState({ activeEditForm: EditFormNames.acquisitionSummary })
+        }
```

Comment:
If I'm understanding this logic well, it would take the user to the edit summary form after finishing with adding/removing properties. I believe the pattern set in Research file is to return to read only view, right?

Something like this would do the trick (I think)

```tsx
setContainerState({
  isEditing: false,
  activeEditForm: undefined
})
```